src/main_WorldViewer.cpp

```diff
@@ -22,18 +22,39 @@ class Config : public REGoth::EngineConfig
   virtual void registerCLIOptions(cxxopts::Options& opts) override
   {
     opts.add_options()
-      ("w,world", "World name to load", cxxopts::value<bs::String>(world), "[NAME]")
+      ("w,world", "Name of the world to load", cxxopts::value<bs::String>(world), "[NAME]")
       ;
   }
 
+  virtual void verifyCLIOptions() override
+  {
+    // Make sure that world is set.
+    if (world.empty())
+    {
+      REGOTH_THROW(InvalidStateException, "World cannot be empty.");
+    }
+
+    // Unify possible inputs.
+    bs::StringUtil::toUpperCase(world);
+    if (not bs::StringUtil::endsWith(world, ".ZEN"))
```

Comment:
Since when does `not` work? :open_mouth:  Is the standard c++ and good practice? Reads nicely, but it does raise eyebrows...

Reply:
@ataulien https://en.wikipedia.org/wiki/C_alternative_tokens :)

Reply:
@frabert

Standard compliant: Yes. But not supported by MSVC the last time I checked.

Reply:
@mworchel I just checked with VS2017 and I didn't need to include any header, FWIW

I will abstain from giving a judgment on the style, as I am quite uncertain about it, but it is at least working.

Reply:
Hm I think the last time I encoutered it was with vs2015 in some uni assignment template and I remember I had to replace all occurences of 'and' and 'or' for it to work. I would probably use '!' just to be consistent with the style of this project and many others but it's not my decision :D

Reply:
I've always done this because I think it's way more readable, and I like writing *human readable* code. I've never touched a Microsoft compiler though, I didn't know it doesn't work there, but I do know that it is standard conform.

Doesn't matter in the end, I will of course change it to fit the general code style. I just wrote it out of habit (I also do this with `and`, `xor`, `or`, ... instead of `&&`, `^`, `||`, ... :D )